### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by scjimenez**
Hi hi, 
as we have another support channel, I changed the Help section in here.

Best, Stephi

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,16 +20,15 @@ Help
 Looking for more help?
 
 - Read `the hosted documentation`_
-- Check `StackOverflow`_ for common problems
```
Comment (mechanomi) [inline README.rst]: I think this line should be left in

Comment (scjimenez) [reply README.rst]: No, it's fine like so. But forgot to delete the stackoverflow link and with your question you draw my attention to that - thanks :)

File: README.rst [hunk 2]
```diff
@@ -20,18 +20,16 @@ Help
...
-- Chat with us on `Slack`_
+- Chat with us via our `support channel`_
```
Comment (MarkusH) [inline README.rst]:
```suggestion
- Chat with us via our `support channels`_
```

**Comment by MarkusH** [comment]
> Thanks @MarkusH, wasn't a typo, it should say "channel" please. We have one channel, Discourse, but out of some reason decided to link to our support page on the website, not on the channel itself. Thanks for the review!

Alright. In that case you need to update the link definition, though (line 31/32). Instead of
```rst
.. _support channels: https://crate.io/support/
```
you need
```rst
.. _support channel: https://crate.io/support/
```

**Comment by scjimenez** [comment]
@MarkusH Nooow I got it - thanks :)

**Pull request merged**